src/scene/scene.js

```diff
@@ -116,6 +120,12 @@ function Scene() {
     this._skyboxIntensity = 1;
     this._skyboxMip = 0;
 
+    this._skyboxRotation = new Quat();
+    this._skyboxRotationMatrix = new Mat3();
+    this._skyboxRotationMatrix.data[0] *= -1.0;
```

Comment:
should this be here?

Reply:
I missed that - thanks - I think I can also make the matrix lazily allocated

Reply:
done - thanks